File: Source/Rio.Web/src/app/shared/models/trade-activity-by-month-dto.ts
export class TradeActivityByMonthDto {
    GroupingDate: string;
    TradeVolume: number;
    NumberOfTrades: number;
    MaximumPrice: number;
    MinimumPrice: number;
    AveragePrice: number;

    constructor(obj?: any) {
        Object.assign(this, obj);
    }
}

File: Rio.Models/DataTransferObjects/TradeActivityByMonthDto.cs
using System;
using Newtonsoft.Json;

namespace Rio.Models.DataTransferObjects
{
    public class TradeActivityByMonthDto
    {
        public DateTime GroupingDate { get; set; }
        public decimal TradeVolume{ get; set; }
        public decimal MinimumPrice{ get; set; }
        public decimal MaximumPrice{ get; set; }
        public decimal AveragePrice{ get; set; }
        public int NumberOfTrades{ get; set; }
    }
}